packages/pwa/app/pages/checkout/index.test.js

```diff
@@ -586,7 +586,7 @@ test('Can proceed through checkout as registered customer', async () => {
     user.click(placeOrderBtn)
 
     await waitFor(() => {
-        expect(window.location.pathname).toEqual(getPathname('/checkout/confirmation'))
+        expect(window.location.pathname).toEqual('/uk/en-GB/checkout/confirmation')
```

Comment:
Because the `createPathWithDefaults` is used to create the pathname for `window.location.pathname`, it does not make sense to assert its result to be something it created.